source/globalVars.py

```diff
@@ -1,35 +1,61 @@
-#globalVars.py
-#A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2007 NVDA Contributors <http://www.nvda-project.org/>
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2006-2022 NVDA Contributors <http://www.nvda-project.org/>
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
 """global variables module
 @var foregroundObject: holds the current foreground object. The object for the last foreground event received.
-@type foregroundObject: L{NVDAObjects.NVDAObject}
-  @var focusObject: holds the current focus object
-@type focusObject: L{NVDAObjects.NVDAObject}
+@var focusObject: holds the current focus object
 @var mouseObject: holds the object that is at the position of the mouse pointer
-@type mouseObject: L{NVDAObjects.NVDAObject}
 @var mouseOldX: the last x coordinate of the mouse pointer before its current position
 @type oldMouseX: int
 @var mouseOldY: the last y coordinate of the mouse pointer before its current position
 @type oldMouseY: int
-  @var navigatorObject: holds the current navigator object
-@type navigatorObject: L{NVDAObjects.NVDAObject}
-@var navigatorTracksFocus: if true, the navigator object will follow the focus as it changes
-@type navigatorTracksFocus: boolean
+@var navigatorObject: holds the current navigator object
 """
- 
+
+import argparse
+import os
+import typing
+
+if typing.TYPE_CHECKING:
+	import NVDAObjects  # noqa: F401 used for type checking only
+
+
+class DefautAppArgs(argparse.Namespace):

```

Comment:
I don't think introducing this code into RC is a good idea. Providing the dummy implementation of `appArgs` causes #13094 so to fix this you would also need to backport #13099 which seems an unnecessary risk.

Reply:
I'm assuming this was not intentional, and that it was accidentally pulled in due to a bad merge conflict fix. 
This code should definitely be removed.

Reply:
Correct. I've rebased and fixed the cherry-pick.

I've also removed the files with changes that consist of unnecessary typing additions.